### PR TITLE
Local run fixes

### PR DESCRIPTION
**Pull request by icemachined**
### What's done:

- fix agent-setting for backend and orchestrator url
- fix saveAgent classpath

### CODE REVIEW

File: save-backend/build.gradle.kts [hunk 1]
```diff
@@ -107,8 +107,8 @@ dependencies {
                 "save-agent, please test them on Linux " +
                 "or put the file with name like `save-agent-*-distribution.jar` built on Linux into libs subfolder."
         )
-        runtimeOnly(files("$buildDir/agentDistro").apply {
-            builtBy(downloadSaveAgentDistroTaskProvider)
+        runtimeOnly(fileTree("$buildDir/agentDistro").apply {
```
Comment (petertrr) [inline save-backend/build.gradle.kts:110]: IIRC @0x6675636b796f75676974687562 has changed this from `fileTree` to `files`.

Comment (icemachined) [reply save-backend/build.gradle.kts:110]: this doesn't work on windows

File: save-orchestrator/src/main/resources/application-dev.properties [hunk 2]
```diff
@@ -1,2 +1,3 @@
 orchestrator.backend-url=http://localhost:5800/internal
-orchestrator.agent-settings.backend-url=http://host.docker.internal:5800/internal
\ No newline at end of file
+orchestrator.agent-settings.backend-url=http://host.docker.internal:5800
+orchestrator.agent-settings.orchestrator-url=http://host.docker.internal:5100
```
Comment (nulls) [inline save-orchestrator/src/main/resources/application-dev.properties:3]: please check the latest changes -- it was changed to heartbeat url

**Review by nulls — CHANGES_REQUESTED** [review]

File: save-orchestrator/src/main/resources/application-dev.properties (hunk 2 above)
Comment (icemachined) [reply save-orchestrator/src/main/resources/application-dev.properties]: fixed

**Review by akuleshov7 — APPROVED** [review]

File: save-backend/build.gradle.kts (hunk 1 above)
Comment (akuleshov7) [reply save-backend/build.gradle.kts:110]: @0x6675636b796f75676974687562

**Review by nulls — APPROVED** [review]

**Review by 0x6675636b796f75676974687562 — APPROVED** [review]

**Pull request merged**